### ISSUE TITLE
Attempt to read property "f" on null

### ISSUE THREAD

**Issue by technghiath** [PHPOffice/PhpSpreadsheet#2677]
This is:

```
- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
- Not appear ErrorException

### What is the current behavior?
- I received an ErrorException, but I don't know which cell it is from from the excel template file
![image](https://user-images.githubusercontent.com/23444133/158302109-d155b8e0-3bdb-4010-af15-ce2736642852.png)

### What are the steps to reproduce?
I would like to create PR fix as below:
```
....
$node->addAttribute('sqref', $item->children('xm', true)->sqref ?? null);
$node->addChild('formula1', $item->formula1->children('xm', true)->f ?? null);
....
```
I'm worried, will this affect performance?

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```
...src\PhpSpreadsheet\Reader\Xlsx.php
....
    /**
     * Loads Spreadsheet from file.
     */
    public function load(string $filename, int $flags = 0): Spreadsheet
    {
       ...
        $node->addAttribute('sqref', $item->children('xm', true)->sqref);
        $node->addChild('formula1', $item->formula1->children('xm', true)->f);
      ...
    }
```
https://github.com/PHPOffice/PhpSpreadsheet/blob/master/src/PhpSpreadsheet/Reader/Xlsx.php#L901-L902

**Comment by oleibman** [comment]
Namespacing logic appears to be a remnant, using prefix rather than namespace. To be corrected by #3078.

**Comment by technghiath** [comment]
@oleibman 
Thanks for helping to solve this issue.
